nifi-registry/nifi-registry-core/nifi-registry-properties/src/main/java/org/apache/nifi/registry/properties/NiFiRegistryProperties.java

```diff
@@ -479,4 +483,25 @@ public String getOidcClaimIdentifyingUser() {
         return getProperty(SECURITY_USER_OIDC_CLAIM_IDENTIFYING_USER, "email").trim();
     }
 
+    /**
+     * Returns the network interface list to use for HTTPS. This method returns a mapping of
+     * network interface property names to network interface names.
+     *
+     * @return the property name and network interface name of all HTTPS network interfaces
+     */
+    public Map<String, String> getHttpsNetworkInterfaces() {
```

Comment:
As the only usage of this method does not use the property names, it looks like this could be simplified to just return a `Set<String>` of interface names that are not blank.
```suggestion
    public Set<String> getHttpsNetworkInterfaceNames() {
```

Reply:
Will simplify